### ISSUE TITLE
Suggestion/Request - Install Selection Option..

### ISSUE THREAD

**Issue by ijbranch** [EtheaDev/SVGIconImageList#288]
Hi Team,
I really love the new v 4.4 installer.  Makes life so much easier.
Unless they are co-dependant, it would be nice if the option was there to select just the VCL components for installation.
I never, and don't anticipate ever doing any FMX work.
If selectable, then at least one or the other, but not both, would need to be compulsory.

Regards,
Ian

**Comment by luebbe** [comment]
Same reason here. I don't need FMX, so I don't install it. As long as the .dpk files are still there, I'm happy to install from the sources.

**Comment by carloBarazzetta** [comment]
Yes: manual installation is always available. I have provided automatic installation for many developers who don't know how to use Packages, Git, etc... to provide all "lazy" developers with an easy way to install components... The new documentation site is also designed for give a simpler tool to use rather than the GitHub wiki...

**Comment by ijbranch** [comment]
Hi Carlo,
**Your work and efforts are greatly appreciated.**  
No, I have no issues whatsoever with the new installer.  What you have done/created is far preferred than GetIt.
As you have said, manual installation is an option, as manual unticking and removal of the FMX package.   once installed.  Which I have done.
Yes, I am a 'lazy' developer, hence the suggestion.   I will pull my head in now.  ;-)
Regards,
Ian

**Comment by carloBarazzetta** [comment]
I'm searching a way to avoid installing packages for firemoney...

**Comment by carloBarazzetta** [comment]
I'll fix it in the next release...
